TFLite/src/main.py

```diff
@@ -0,0 +1,56 @@
+import TFLiteParser
+import os
+
+# Function to calculate duplication statistics op operators i.e.
+# number of unique tensors and operators accross models 
+def get_duplication(tflite_parser):
+    node_vis = dict()
+    edge_vis = dict()
+
+    total_nodes = 0
+    duplicated_nodes = 0
+
+    total_edges = 0
+    duplicated_edges = 0
+
+    for file in os.listdir("models/"):
+        graph = tflite_parser.parse_graph("models/" + file)
+
+        # Calculating number of of total nodes and duplicated nodes in each graph
+        total_nodes += len(graph.nodes)
+        total_edges += len(graph.get_traversed_edges())
+
+        for node in graph.nodes:
+            key = node.serialize()
+
+            if key not in node_vis:
```

Comment:
nit: node_visited. Same for other variables.

Reply:
Fixed in later PR.